includes/class-kadence-blocks-prebuilt-library-rest-api.php

```diff
@@ -2007,6 +2100,16 @@ public function get_collection_params() {
 			'type'              => 'array',
 			'sanitize_callback' => array( $this, 'sanitize_image_sizes_array' ),
 		);
+		$query_params[ self::PROP_EVENT_LABEL ] = array(
+			'description'       => __( 'The Event Label', 'kadence-blocks' ),
+			'type'              => 'string',
+			'sanitize_callback' => 'sanitize_text_field',
+		);
+		$query_params[ self::PROP_EVENT_DATA ] = array(
```

Comment:
As mentioned above, you can make this take an array/an array of objects [schema](https://github.com/stellarwp/prophecy-wordpress/blob/735a5281e92ee8ca43c4860c8dd2c453fa5e7200/src/Rest/Content/Controllers/Content.php#L162-L181).